src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+
+import { UserInfo } from '../UserInfo';
+import { ITodoInfo } from '../../types/ITodoInfo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: ITodoInfo;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const { title, completed, id, user } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={`TodoInfo TodoInfo${completed ? `--completed` : ``}`}
```

Comment:
`&&` can be used instead. Also, better use classnames tool for conditional classes
https://www.npmjs.com/package/classnames